app/scripts/lib/transaction/smart-transactions.ts

```diff
@@ -123,6 +125,8 @@ class SmartTransactionHook {
     this.#isDapp = transactionMeta.origin !== ORIGIN_METAMASK;
     this.#chainId = transactionMeta.chainId;
     this.#txParams = transactionMeta.txParams;
+    this.#shouldShowStatusPage =
+      transactionMeta.type !== TransactionType.bridge;
```

Comment:
Is this needed for `TransactionType.bridgeApproval` as well?

Reply:
Good catch, I thought I was already skipping STX if it was `bridgeApproval`. I added that code back in [c577043](https://github.com/MetaMask/metamask-extension/pull/28460/commits/c57704325b369653ed235ffe2df2f384549bf43c)